tokens/sys/breakpoint.json

```
@@ -0,0 +1,29 @@
+{
+  "breakpoints": {
+    "zero": {
+      "value": "0rem",
```

Let's make this "0", which matches what we do for space tokens.
```suggestion
      "value": "0",
```